apps/api/src/app/logs/usecases/create-log/create-log.usecase.ts

```diff
@@ -7,7 +7,7 @@ export class CreateLog {
   constructor(private logRepository: LogRepository) {}
 
   async execute(command: CreateLogCommand): Promise<LogEntity> {
-    let rawData: string = null;
+    let rawData: string | null = null;
```

Comment:
```suggestion
    let rawData = '';
```
This should fix it and should be ok. If we don't want to risk it we can set as:
```
let rawData;
```
and still will be ok and warning free. Just checked it.

Reply:
I've tried to not touch the code itself, but rather the types, that's why I didn't assign an empty string to it.
If you do `let rawData;` then the type is `any` and you can assign whatever you want. The code below `JSON.stringify` assigns the string, that's why the appropriate type is `let rawData: string | null = null;`.

Reply:
I truly thought TS would interpret it as `undefined` type. Wow, my bad. 😓 

I know I am insisting a bit on the issue in a different commands but not keen on using `null`. It is not directly passing to `JSON.stringify` but `JSON.stringify(null)` becomes `'null'` while `JSON.stringify(undefined)` becomes `undefined`. In this case it would be in the `command.raw` but if arrives to `rawData` we can have problems thinking that could get an empty value while it could hold null stringified.
Just wanted to highlight those risks of using `null` because they can lead to unexpected bugs. So apologies for the repetition. 🙏🏻

Reply:
I've checked DB and I don't see strings like `"null"`, but I see a lot `raw: null`, but you're right it's better to change it to undefined here, I will do that